tests/unit/store-test.js

```diff
@@ -325,6 +325,32 @@ test("clear will destroy everything for a given type", function() {
   equal(catsAfter.get("length"), 1);
 });
 
+test("clearAll will destroy everything (this kills all the kittens and people)", function() {
+  store.push("person", {
+    id: 9,
+    firstName: "Brandon",
+    lastName: "Williams",
+    cat_id: 1
+  });
+
+  store.push("person", {
+    id: 8,
+    firstName: "Toran",
+    lastName: "Billups",
+    cat_id: 1
+  });
+
+  store.push("cat", {
+    id: 1,
+    color: "red"
+  });
+
+  store.clearAll();
+
+  equal(store.find("person").get("length"), 0);
```

Comment:
would you mind adding a "pre clear" assertion to show you get 1 length before the clear? It helps tell the story (before and after) like :)